index.js

```diff
@@ -4,6 +4,12 @@ const fs = require('fs')
 const path = require('path')
 const EE = require('events').EventEmitter
 const Minimatch = require('minimatch').Minimatch
+const mmopt = {
```

Comment:
Reusing the same minimatch options object will have some side effects.  It _should_ be fine, but just in case, it should be left out of this commit, since it's unrelated.

Reply:
moved most back